<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditHostname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>172.21.28.65</string>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>301</width>
      <height>181</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabText">
     <attribute name="title">
      <string>Text</string>
     </attribute>
     <widget class="QLineEdit" name="lineEditText">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>100</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Text :</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxScroll">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>78</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Scroll</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxCenter">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>78</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Center</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxScrollDelay">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="prefix">
       <string>Delay: </string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxBigFont">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>78</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>bigFont</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPosition">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Position</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxPosition_X">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="prefix">
       <string>x: </string>
      </property>
      <property name="maximum">
       <number>31</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxPosition_Y">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="prefix">
       <string>y: </string>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabClock">
     <attribute name="title">
      <string>Clock</string>
     </attribute>
     <widget class="QCheckBox" name="checkBoxShowClock">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Clock</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxAktiv">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Clock or Date</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxSeconds">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Seconds</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxWeek">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Display week</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxSwitchSec">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="prefix">
       <string>Switch Sec: </string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxStyle">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelBrightness">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="text">
        <string>Brightness : 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderBrightness">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnColor">
       <property name="text">
        <string>Set Color</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEditTopic">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>lafayette/207</string>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="subscribeBtn">
    <property name="geometry">
     <rect>
      <x>419</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Subscribe</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>211</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Status :</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Brocker :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelStatus">
       <property name="text">
        <string>disconnected</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Matrix Info :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QTextBrowser" name="textBrowserMatrixInfo"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>556</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnColor</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>colorBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderBrightness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>brightnessChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxScroll</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxCenter</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxScrollDelay</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxPosition_X</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>positionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxPosition_Y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>positionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditText</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxBigFont</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subscribeBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>MQTTsubscribe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxShowClock</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateClock()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxSeconds</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateClock()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAktiv</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateClock()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxWeek</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateClock()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSwitchSec</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateClock()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>positionChanged()</slot>
  <slot>colorBtnClicked()</slot>
  <slot>brightnessChanged()</slot>
  <slot>updateText()</slot>
  <slot>bitmapDraw()</slot>
  <slot>MQTTsubscribe()</slot>
  <slot>updateClock()</slot>
 </slots>
</ui>
